build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -364,7 +370,7 @@ CONFIG_DEV_CONSOLE=y
 # CONFIG_FDCLONE_DISABLE is not set
 # CONFIG_FDCLONE_STDIO is not set
 CONFIG_SDCLONE_DISABLE=y
-CONFIG_NFILE_DESCRIPTORS=8
+CONFIG_NFILE_DESCRIPTORS=64
 CONFIG_NFILE_STREAMS=8
```

Comment:
Maybe you would need to increase the File Streams too accordingly?

Reply:
Umm could you please tell how do the file streams get modified in accordance with descriptors ?

Reply:
Umm could you please tell how do the file streams get modified in accordance with descriptors ?

Reply:
fd = open("path"); opens a file via a file descriptor.
Whereas, file = fopen(<>) opens the file as a file stream.

What you modified changes the former, but not the later.
So you can change according to your requirements,

Reply:
8 is using in our use case. Let's keep 8.